### PR TITLE
Doc trait+object (enum) serialization for Jackson

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

File: akka-serialization-jackson/src/test/scala/doc/akka/serialization/jackson/SerializationDocSpec.scala [hunk 1]
```diff
@@ -119,5 +127,103 @@ object SerializationDocSpec {
     #//#whitelist-class-prefix
   """
 
+  object CustomAdtSerializer {
```
Comment (raboof) [inline akka-serialization-jackson/src/test/scala/doc/akka/serialization/jackson/SerializationDocSpec.scala]: I wonder if it'd be easier to put the whole example in its own file?

**Review by raboof — APPROVED** [review]
Super useful to have this documented with a full example!

File: akka-serialization-jackson/src/test/scala/doc/akka/serialization/jackson/SerializationDocSpec.scala (hunk 1 above)
Comment (patriknw) [reply akka-serialization-jackson/src/test/scala/doc/akka/serialization/jackson/SerializationDocSpec.scala]: good suggestion, fixed in 94c0586

**Review by chbatey — APPROVED** [review]
LGTM

**Pull request merged**